src/theia/sfm/feature_extractor_and_matcher.h

```diff
@@ -92,9 +92,10 @@ class FeatureExtractorAndMatcher {
   bool AddImage(const std::string& image_filepath,
                 const CameraIntrinsicsPrior& intrinsics);
 
-  // Sort and fill the vector of masks in order to make a correspondance between
-  // the idexes of 'image_filepaths_' and 'mask_filepaths_'.
-  bool SetMasksForFeaturesExtraction(std::vector<std::string> mask_filepaths);
+  // Add a mask to the 'image_masks_' map.
```

Comment:
Details of the internal variables should be hidden from the caller in the public API. Change this comment to explain that the mask indicates the valid area for keypoint extraction, what values should be given in the mask, etc.